tests/dem/integration_schemes_accuracy.output

```diff
@@ -1,10 +1,4 @@
+DEAL::Explicit Euler is a 1 order integration scheme
+DEAL::Velocity Verlet is a 2 order integration scheme
+DEAL::Gear3 is a 5 order integration scheme
```

Comment:
So much better :)

Reply:
It would be better to leave the actual double values instead of rounding. Or maybe give both values (the ratio and the rounded order). This system is deterministic so the result won't be compiler dependent, it's not like DEM.